modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/filters/modals/selection_filter/SelectionFilter.tsx

```diff
@@ -456,13 +468,33 @@ function Body({
 								aria-label={Liferay.Language.get(
 									'choose-an-option'
 								)}
+								disabled={!!filter}
```

Comment:
We're removing a perfectly fine feature just because of implementation? :disappointed:  If this is true, it means implementation is terrible and we need to refactor it as soon as possible.